package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "prisonstaffhub",
```

Comment:
This I'm pretty sure will change the output roleName in insights, I will test when its on dev